horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/internal/services/UserService.java

```diff
@@ -47,6 +47,11 @@ public interface UserService {
    @Blocking
    void createUser(@RequestBody(required = true) NewUser user);
 
+   @DELETE
+   @Path("removeUser")
+   @Blocking
+   void removeUser(@RequestBody(required = true) String username);
```

Comment:
In a restful API, the delete method should "delete the resources (identified by the Request-URI)." So a call should be 
DELETE http://exmaple.com/api/user/anon

```suggestion
   @DELETE
   @Path("{username}")
   @Blocking
   void removeUser(@PathParam(name = "username", required = true) String username);
```